client/jetpack-cloud/sections/agency-dashboard/sites-overview/site-preview-pane/site-preview-pane-header/style.scss

```diff
@@ -1,59 +1,108 @@
+			background-size: cover;
+			background-repeat: no-repeat;
+			height: 100px;
+		}
+
+		.site-preview__header-favicon {
+			position: relative;
+			top: -44px;
+			left: 48px;
+			width: 64px;
+			height: 64px;
+			margin-bottom: -64px;
+		}
+
+		.site-preview__header-content {
+			padding: 48px 48px 24px 48px;
+			display: flex;
+			align-items: center;
 
-	.site-preview__header-favicon {
-		position: relative;
-		top: -44px;
-		left: 48px;
-		width: 64px;
-		height: 64px;
-		margin-bottom: -64px;
+			.site-preview__header-title-summary {
+				flex-grow: 1;
+				display: flex;
+				flex-direction: column;
+
+				.site-preview__header-title {
+					font-style: normal;
+					font-weight: 500;
+					font-size: rem(32px);
+					line-height: 32px;
+					color: var(--studio-black);
+					margin-bottom: 4px;
+				}
+
+				.site-preview__header-summary {
+
+					.site-preview__header-summary-link {
```

Comment:
Although there's nothing wrong with nesting selectors, then my own indentation OCD starts to trigger slightly 😝 

Are we purposefully doing this to create stronger selectors to override styling from the Gutenberg component or similar?

Reply:
Yes, you are right, I had to create stronger selectors in many places.

Reply:
Yep, that's right, using the selector specificity to avoid `!import` as much as possible.